### ISSUE TITLE
Sort doesn't work...

### ISSUE THREAD

**Issue by amanvm** [okbob/pspg#224]
Sort doesn't seem to work on numeric(10,7) column for longitude in attached US Census by county data from 2019.

[us_counties_pop_est_2019.csv](https://github.com/okbob/pspg/files/11878567/us_counties_pop_est_2019.csv)

System information:
- macOS Ventura 3.4.1 (22F82)
- AARCH64 (ARM architecture)
- psql (PostgreSQL) 15.3 (Homebrew)
- pgcli version: 3.5.0
- pspg version: 5.7.7

Steps to reproduce:
```
CREATE TABLE us_counties_pop_est_2019 (
    state_fips text,
    county_fips text,
    region smallint,
    state_name text,
    county_name text,
    area_land bigint,
    area_water bigint,
    internal_point_lat numeric(10,7),
    internal_point_lon numeric(10,7),
    pop_est_2018 integer,
    pop_est_2019 integer,
    births_2019 integer,
    deaths_2019 integer,
    international_migr_2019 integer,
    domestic_migr_2019 integer,
    residual_2019 integer,
    CONSTRAINT counties_2019_key PRIMARY KEY (state_fips, county_fips)
);
\copy us_counties_pop_est_2019 FROM '<path_to_us_counties_pop_est_2019.csv>' WITH (FORMAT CSV, HEADER);
table us_counties_pop_est_2019
```

Then select the column for internal_point_lon, and press ``d`` or ``a``, and scroll to see it for yourself.

**Comment by okbob** [comment]
should be fixed by 4a71c13, please check

**Comment by amanvm** [comment]
@okbob Yes that fixed it, thanks!

**Comment by okbob** [comment]
út 27. 6. 2023 v 21:20 odesílatel amanvm ***@***.***> napsal:

> Closed #224 <https://github.com/okbob/pspg/issues/224> as completed.
>

thank you for bug report
